### DISCUSSION TITLE
Can Flux store the generated artifacts back in git as release artifacts

### DISCUSSION THREAD

**Discussion by mrmartan** [fluxcd/flux2#3891] [Q&A]
Flux stores artifacts temporarily locally in the controller, such as `stored artifact for revision 'master@sha1:4b531345322404217601a2c7db3f5e0ea213e875'`. Can it be configured to upload that artifact back to git, lets say as release artifacts, or in any other way?

**Comment by hiddeco** [comment]
It can not, but the source-controller can be configured to make use of a persistent storage to ensure the artifacts survive a Pod restart. See: https://fluxcd.io/flux/cheatsheets/bootstrap/#persistent-storage-for-flux-internal-artifacts

**Reply by mrmartan** [reply]
Do you think it would be a good feature request?

**Reply by hiddeco** [reply]
It would not provide any use to Flux itself, so not very likely.
